.circleci/config.yml

```diff
@@ -0,0 +1,58 @@
+# Python CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-python/ for more details
+#
+version: 2
+jobs:
+  build:
+    docker:
+      # specify the version you desire here
+      # use `-browsers` prefix for selenium tests, e.g. `3.6.1-browsers`
+      - image: circleci/python:3.6.1
+
+      # Specify service dependencies here if necessary
+      # CircleCI maintains a library of pre-built images
+      # documented at https://circleci.com/docs/2.0/circleci-images/
+      # - image: circleci/postgres:9.4
+
+    working_directory: ~/repo
+
+    steps:
+      - checkout
+
+      # Download and cache dependencies
+      - restore_cache:
+          keys:
+          - v1-dependencies-{{ checksum "test_requirements.txt" }}
+          # fallback to using the latest cache if no exact match is found
+          - v1-dependencies-
+
+      - run:
+          name: install dependencies
+          command: |
+            python3 -m venv venv
```

Comment:
@thraxil Do you think we could run the test with python 3 and 2 as well? This app is installed in AMC which it's runs python 3 and is installed in edx-platform as well, which it's runs python 2. So we should make always sure that the app works with both versions of python.

// cc @bezidejni

Reply:
@melvinsoft Good point, agreed.

Reply:
No problem (I also know how to do that easier with Travis anyway...)